specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/cluster.json

```diff
@@ -916,13 +1028,18 @@
         "minInstanceCount": {
           "type": "integer",
           "format": "int32",
-          "description": "The minimum instance count of the cluster."
+          "description": "The minimum instance count of the cluster.",
+          "x-ms-secret": true
```

Comment:
Double checking: Is min count a secret?

Reply:
> Double checking: Is min count a secret?

`x-ms-secret` does not mean it is secret, the "secret" word is confused for us. It is in order to fix the S360 issues that this property is shown in PUT request body, but the GET response doesn't contain it.